### ISSUE TITLE
(Fixed for OP) Charmonium-Forge-1.16.4-2.1.3 data pack validation failed

### ISSUE THREAD

**Issue by StygianEmperor** [svenhjol/Charmonium#8]
**Versions**
* Charm - 2.1.3
* Minecraft - 1.16.4
* Loader - Forge 35.1.7

**Description**
When trying to run Minecraft with Charmonium-Forge-1.16.4-2.1.3, validation fails when hitting "Done" to close the data pack window when creating a new world. Additionally, old worlds prior to using Charmonium won't load while Charmonium's .jar is in the mods folder. Charm-Forge-1.16.4-2.1.3 by itself seems to work fine.

**Comment by BugmanBugman** [comment]
Can confirm this

**Comment by svenhjol** [comment]
Thanks for the report - didn't catch this because it wasn't always occurring. The problem is due to Charm not always being the first to load. Please could you try this Charm hotfix and let me know if the problem resolves: https://github.com/svenhjol/Charm/releases/download/2.1.3-forge-hotfix1/charm-forge-1.16.4-2.1.3-hotfix1.jar

**Comment by BugmanBugman** [comment]
Still seems to be happening for me with the hotfix

**Comment by Alaberti** [comment]
Same here, no change with hotfix

**Comment by svenhjol** [comment]
Thanks for the report and testing. Until I can figure out what's causing this, I have archived the curseforge jar.

**Comment by svenhjol** [comment]
Please try these versions to test the datapack error and let me know if it's resolved:
* https://github.com/svenhjol/Charm/releases/download/2.1.4/charm-forge-1.16.4-2.1.4.jar
* https://github.com/svenhjol/Charmonium/releases/download/2.1.4/charmonium-forge-1.16.4-2.1.4.jar

**Comment by Alaberti** [comment]
No dice boss

**Comment by svenhjol** [comment]
> No dice boss

Please share your startup log

**Comment by BugmanBugman** [comment]
Contrary for me, it fixed it

@Alaberti did you make sure to use both of the files? could be it

**Comment by Alaberti** [comment]
NVM, I forgot mod testing 101 - I was running it in my pack, but when I tested just charm, charmonium, and quark, the world loaded up just fine! Attached are the latest.log files from both runs. For the modpack run, I'm still narrowing down why adding charmonium causes the data pack issue.

Clean run: https://pastebin.com/9MdAXt3f
Pack run: https://drive.google.com/file/d/1NYeGBuLIwNmHqp-fnkS3QarYgnNArCy2/view?usp=sharing (83+ megs, couldn't get it to upload to pastebin, not sure how helpful it actually is)

Because I was in the middle of building the pack when the Overwolf change happened, I don't have an easy way to share it with you if you wanted to do some compatibility tests of your own, but below is a screenshot of the mods folder (50~ mods):
![image](https://user-images.githubusercontent.com/10929796/101064299-aeb5ce80-3548-11eb-981c-91173d081492.png)

**Comment by Alaberti** [comment]
Got it! It will still error when using Pneumaticraft Represurrized. I tested with only autoreglib, charm 2.1.4, charmonium 2.1.4, quark 2.4-277, and pneumaticcraft-repressurized 1.16.4-2.7.0-83.

Latest.log: https://pastebin.com/7g1bCtDy

I additionally verified that it worked with all the other mods in the pack and that the only error, with what I tested, was when penumaticcraft was included with charmonium.
